atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/AtomicFU.common.kt

```diff
@@ -461,11 +461,11 @@ public inline fun AtomicLong.updateAndGet(function: (Long) -> Long): Long {
 public class AtomicIntArray(size: Int) {
     private val array = Array(size) { atomic(0) }
 
-    @JsName("size\$atomicfu")
+    @JsName("size\$atomicfu\$")
```

Comment:
Great job with meticulous JS methods marking!

Unfortunately, it contains a lot of not-so-readable symbols that have not so clear purpose.
It would be nice if we could have a single `.kt` file with all these names in a single place as `internal const val`
and then reference it on the use-site:
```
@JsName(JS_ARRAY_GET)
public operator fun get(index: Int): AtomicInt
```

Then we will have a single place with all the documentation and explanation of what it is and why it is written in this way. As bouns, JVM users won't see it, but curious ones will be able to study it if they want.
WDYT?

Reply:
Moved all mangled names to one file, really looks much better now